doc/api/process.md

```diff
@@ -872,96 +896,97 @@ previous setting of `process.exitCode`.
 added: v2.0.0
 -->
 
-Note: this function is only available on POSIX platforms (i.e. not Windows,
-Android)
-
-Gets the effective group identity of the process. (See getegid(2).)
-This is the numerical group id, not the group name.
+The `process.getegid()` method returns the numerical effective group identity
+of the Node.js process. (See getegid(2).)
 
 ```js
 if (process.getegid) {
   console.log(`Current gid: ${process.getegid()}`);
 }
 ```
 
+*Note*: This function is only available on POSIX platforms (i.e. not Windows
+or Android)
 
 ## process.geteuid()
 <!-- YAML
 added: v2.0.0
 -->
 
-Note: this function is only available on POSIX platforms (i.e. not Windows,
-Android)
-
-Gets the effective user identity of the process. (See geteuid(2).)
-This is the numerical userid, not the username.
+The `process.geteuid()` method returns the numerical effective user identity of
+the process. (See geteuid(2).)
 
 ```js
 if (process.geteuid) {
   console.log(`Current uid: ${process.geteuid()}`);
 }
 ```
 
+*Note*: This function is only available on POSIX platforms (i.e. not Windows or
```

Comment:
it is `undefined` on non-POSIX platforms

https://github.com/nodejs/node/blob/master/src/node.cc#L1826-L2145